ethexe/ethereum/src/lib.rs

```diff
@@ -41,8 +42,10 @@ use alloy::{
 };
 use anyhow::{Result, anyhow};
 use async_trait::async_trait;
-use ethexe_common::{Address as LocalAddress, Digest, ecdsa::PublicKey};
-use ethexe_signer::Signer as LocalSigner;
+use gsigner::secp256k1::{
+    Address as LocalAddress, Digest as LocalDigest, PublicKey as LocalPublicKey,
```

Comment:
I'd rename them from local.

Maybe use them as is, but rename alloy ones: use alloy::Address as AlloyAddress etc